Readme.md

```diff
@@ -13,28 +13,25 @@ Horseman has:
   * built in jQuery for easier page manipulation,
   * built in bluebird for easier in-browser async.
 
-## Example
+## Examples
 
-Search on Google:
+Count Twitter Followers Concurrently
 
 ```js
-var Horseman = require('node-horseman');
-var horseman = new Horseman();
-
-horseman
-  .userAgent('Mozilla/5.0 (Windows NT 6.1; WOW64; rv:27.0) Gecko/20100101 Firefox/27.0')
-  .open('http://www.google.com')
-  .type('input[name="q"]', 'github')
-  .click('button:contains("Google Search")')
-  .keyboardEvent('keypress', 16777221)
-  .waitForSelector('div.g')
-  .count('div.g')
-  .log() // prints out the number of results
-  .close();
-
+const Horseman = require('node-horseman');
+const users = ['PhantomJS', 'nodejs'];
+
+users.forEach((user) => {
```

Comment:
@awlayton Sure. Apparently there was a copy paste error. And I had `horseman.open` on the same line because previously your docs had `horseman.close()` so I was trying to maintain consistency. But you're right, if `.close` is chained, this should be as well.